internal/backend/deployments/deployments.go

```diff
@@ -3,34 +3,87 @@ package deployments
 import (
 	"context"
 	"fmt"
+	"math/rand/v2"
+	"time"
 
 	"go.uber.org/zap"
 
 	"go.autokitteh.dev/autokitteh/internal/backend/auth/authcontext"
 	"go.autokitteh.dev/autokitteh/internal/backend/auth/authz"
+	"go.autokitteh.dev/autokitteh/internal/backend/configset"
 	"go.autokitteh.dev/autokitteh/internal/backend/db"
 	"go.autokitteh.dev/autokitteh/internal/backend/telemetry"
 	"go.autokitteh.dev/autokitteh/sdk/sdkerrors"
 	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
 	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
+type Config struct {
+	// Draining will happen each interval + rand(jitter).
+	// Jitter is to prevent all instances from deactivating at the same time.
```

Comment:
Jitter is relevant when you have agents that are synchronized (e.g. threads in a single process, or synchronized processes) or when the amount of agents is large enough that concurrency is expected most of the time anyway.

This is not the case here. I'm saying using jitter is bad, I'm saying it's solving a problem you shouldn't have anyway.

Reply:
It's just to add another level of randomness in case weird things happen.